install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -113,6 +113,12 @@ spec:
 {{- if .Values.enableProtocolSniffing }}
           - name: PILOT_ENABLE_PROTOCOL_SNIFFING
             value: "{{ .Values.enableProtocolSniffing }}"
+          - name: PILOT_DISABLE_XDS_MARSHALING_TO_ANY
+            value: "1"
```

Comment:
not sure why this is here

Reply:
thanks, me neither, probably copy/paste issue.  Removed